policy/modules/roles/sysadm.te

```diff
@@ -41,6 +41,9 @@ corenet_ib_manage_subnet_unlabeled_endports(sysadm_t)
 dev_read_kmsg(sysadm_t)
 dev_rw_ipmi_dev(sysadm_t)
 
+# Use nfs tools
+fs_rw_nfsd_fs(sysadm_t)
```

Comment:
Seems like this should be in `rpc_admin()` instead.

Reply:
I agree that makes more sense.